GitUI/Translation/English.xlf

```diff
@@ -8709,6 +8709,14 @@ This will just checkout on the submodule the commit determined by the superproje
       </trans-unit>
     </body>
   </file>
+  <file datatype="plaintext" original="Plugin" source-language="en">
+    <body>
+      <trans-unit id="Description.Text">
+        <source>Plugin Manager</source>
+        <target />
+      </trans-unit>
+    </body>
+  </file>
```

Comment:
This must not be committed, it is noted in https://github.com/gitextensions/gitextensions/wiki/Translations

Reply:
The wiki states that this only applies to `English.Plugins.xlf` and not `English.xlf`, or am I missing something here?